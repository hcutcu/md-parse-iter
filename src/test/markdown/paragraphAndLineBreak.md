Here comes a paragraph of text.  
This text is not going be separated by any breaks. Unless has 2 spaces at the end of the line.  
in a paragraph, you can have multiple lines of text.