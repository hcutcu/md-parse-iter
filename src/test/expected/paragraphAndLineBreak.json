[
  {
    "type": "paragraph",
    "children": [
      {
        "type": "span",
        "content": "Here comes a paragraph of text.",
        "attributes": []
      },
      {
        "type": "lineBreak"
      },
      {
        "type": "span",
        "content": "This text is not going be separated by any breaks. Unless has 2 spaces at the end of the line.",
        "attributes": []
      },
      {
        "type": "lineBreak"
      },
      {
        "type": "span",
        "content": "in a paragraph, you can have multiple lines of text.",
        "attributes": []
      }
    ]
  }
]
